qadence2_expressions/__init__.py

```diff
@@ -1,29 +1,9 @@
 from __future__ import annotations
 
+from importlib import import_module
+
 from .collect import collect_operators
-from .core.constructors import (
-    array_parameter,
-    array_variable,
-    function,
-    parameter,
-    parametric_operator,
-    projector,
-    promote,
-    symbol,
-    unitary_hermitian_operator,
-    value,
-    variable,
-)
-from .core.environment import (
-    add_grid_options,
-    add_qpu_directives,
-    add_settings,
-    reset_ir_options,
-    set_grid_scale,
-    set_grid_type,
-    set_number_qubits,
-    set_qubits_positions,
-)
+from .core import *
```

Comment:
imo it's usually better to not use wildcard imports, but since you've specified the __all__ variable it's quite okay here.

Reply:
Yeah I know. In the end, I am just wondering if all that machinery is really necessary for a single submodule.